testscommon/genesisMocks/runTypeComponentsStub.go

```diff
@@ -0,0 +1,118 @@
+package genesisMocks
+
+import (
+	"github.com/multiversx/mx-chain-go/process/block/preprocess"
+	"github.com/multiversx/mx-chain-go/process/coordinator"
+	"github.com/multiversx/mx-chain-go/process/smartContract/hooks"
+	"github.com/multiversx/mx-chain-go/process/smartContract/processProxy"
+	"github.com/multiversx/mx-chain-go/process/smartContract/processorV2"
+	"github.com/multiversx/mx-chain-go/process/smartContract/scrCommon"
+	"github.com/multiversx/mx-chain-go/sharding"
+	"github.com/multiversx/mx-chain-go/state"
+	"github.com/multiversx/mx-chain-go/state/factory"
+	"github.com/multiversx/mx-chain-go/testscommon/enableEpochsHandlerMock"
+	"github.com/multiversx/mx-chain-go/testscommon/hashingMocks"
+	"github.com/multiversx/mx-chain-go/testscommon/marshallerMock"
+)
+
+// RunTypeComponentsStub -
+type RunTypeComponentsStub struct {
```

Comment:
Not another stub, please.
Use another existing one.  If any cylcic imports, try to isolate those packages

Reply:
cannot use existing stub because of cyclic imports